apps/web_delivery/config/config.exs

```diff
@@ -18,6 +13,10 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:request_id]
 
+config :rollbax,
+  access_token: System.get_env("ROLLBAR_ACCESS_TOKEN"),
+  environment: System.get_env("APPLICATION_ENVIRONMENT") || Mix.env
```

Comment:
Because `MIX_ENV` would be `prod` instead of `production`. And the deploy notification hook for rollbar specifies `production` by default. It just keeps it more consistent with the other projects in rollbar. I'm happy to do whichever though.